--- a/cmake/paths_compilation_installation.cmake	2026-01-30 19:43:07.000000000 +0100
+++ b/cmake/paths_compilation_installation.cmake	2026-03-17 06:21:43.354359946 +0100
@@ -20,19 +20,19 @@
 
 if(UNIX AND NOT APPLE)
   # for Linux, BSD, Solaris, Minix
-  set(COMPIL_PREFIX               "/usr/local/")
-  set(INSTALL_PREFIX              "/usr/local/")
+  set(COMPIL_PREFIX               "/usr/")
+  set(INSTALL_PREFIX              "/usr/")
   set(QET_BINARY_PATH             "bin/")
   set(QET_COMMON_COLLECTION_PATH  "share/qelectrotech/elements/")
   set(QET_COMMON_TBT_PATH         "share/qelectrotech/titleblocks/")
   set(QET_LANG_PATH               "share/qelectrotech/lang/")
   set(QET_EXAMPLES_PATH           "share/qelectrotech/examples/")
-  set(QET_LICENSE_PATH            "doc/qelectrotech/")
-  set(QET_MIME_PACKAGE_PATH       "../share/mime/packages/")
+  set(QET_LICENSE_PATH            "share/doc/qelectrotech/")
+  set(QET_MIME_PACKAGE_PATH       "share/mime/packages/")
   set(QET_DESKTOP_PATH            "share/applications/")
   set(QET_ICONS_PATH              "share/icons/hicolor/")
-  set(QET_MAN_PATH                "man/")
-  set(QET_APPDATA_PATH            "share/appdata")
+  set(QET_MAN_PATH                "share/man/")
+  set(QET_APPDATA_PATH            "share/metainfo")
 endif()
 
 if(APPLE)
