diff --git a/CMakeLists.txt b/CMakeLists.txt
index b1433e64..e531a042 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -25,6 +25,20 @@ if(DEBUG)
        set(CMAKE_VERBOSE_MAKEFILE ON)
 endif()

+if (${CMAKE_VERSION} VERSION_GREATER "3.0.0" AND CMAKE_MAKE_PROGRAM MATCHES "ninja")
+  set(MONERO_GUI_PARALLEL_COMPILE_JOBS "" CACHE STRING "The maximum number of concurrent compilation jobs.")
+  if (MONERO_GUI_PARALLEL_COMPILE_JOBS)
+    set_property(GLOBAL APPEND PROPERTY JOB_POOLS compile_job_pool=${MONERO_GUI_PARALLEL_COMPILE_JOBS})
+    set(CMAKE_JOB_POOL_COMPILE compile_job_pool)
+  endif ()
+
+  set(MONERO_GUI_PARALLEL_LINK_JOBS "" CACHE STRING "The maximum number of concurrent link jobs.")
+  if (MONERO_GUI_PARALLEL_LINK_JOBS)
+    set_property(GLOBAL APPEND PROPERTY JOB_POOLS link_job_pool=${MONERO_GUI_PARALLEL_LINK_JOBS})
+    set(CMAKE_JOB_POOL_LINK link_job_pool)
+  endif ()
+endif()
+
 set(BUILD_GUI_DEPS ON)
 set(ARCH "x86-64" CACHE STRING "Target architecture")
 set(BUILD_64 ON CACHE BOOL "Build 64-bit binaries")
