diff --git a/cmake/VersionGui.cmake b/cmake/VersionGui.cmake
index e225bf2b..1f8574e8 100644
--- a/cmake/VersionGui.cmake
+++ b/cmake/VersionGui.cmake
@@ -31,17 +31,7 @@ function (write_static_version_header tag)
   configure_file("${CMAKE_CURRENT_SOURCE_DIR}/src/version.js.in" "${CMAKE_CURRENT_SOURCE_DIR}/version.js")
 endfunction ()

-find_package(Git QUIET)
-if ("$Format:$" STREQUAL "")
-  # We're in a tarball; use hard-coded variables.
-  write_static_version_header("release")
-elseif (GIT_FOUND OR Git_FOUND)
-  message(STATUS "Found Git: ${GIT_EXECUTABLE}")
-  get_version_tag_from_git("${GIT_EXECUTABLE}")
-  write_static_version_header(${VERSIONTAG})
-else()
-  message(STATUS "WARNING: Git was not found!")
-  write_static_version_header("unknown")
-endif ()
+write_static_version_header("release")
+
 add_custom_target(genversiongui ALL
     DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/version.js")
