diff --git a/monero/contrib/epee/include/storages/portable_storage.h b/monero/contrib/epee/include/storages/portable_storage.h
index 1e68605ab..801bb2c34 100644
--- a/monero/contrib/epee/include/storages/portable_storage.h
+++ b/monero/contrib/epee/include/storages/portable_storage.h
@@ -40,6 +40,8 @@
 #include "span.h"
 #include "int-util.h"

+#include <boost/mpl/contains.hpp>
+
 namespace epee
 {
   namespace serialization

