diff --git a/cmake/VersionGui.cmake b/cmake/VersionGui.cmake
index 960c3f4e..650fe229 100644
--- a/cmake/VersionGui.cmake
+++ b/cmake/VersionGui.cmake
@@ -30,20 +30,7 @@ function (write_static_version_header VERSION_TAG_GUI)
   configure_file("${CMAKE_CURRENT_SOURCE_DIR}/src/version.js.in" "${CMAKE_CURRENT_SOURCE_DIR}/version.js")
 endfunction ()

-find_package(Git QUIET)
-if ("$Format:$" STREQUAL "")
-  # We're in a tarball; use hard-coded variables.
-  write_static_version_header("release")
-elseif (GIT_FOUND OR Git_FOUND)
-  message(STATUS "Found Git: ${GIT_EXECUTABLE}")
+write_static_version_header("release")

-  include(GitGetVersionTag)
-  git_get_version_tag(${GIT_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR} VERSION_TAG_GUI)
-  STRING(REGEX REPLACE "^v([0-9])" "\\1" VERSION_TAG_GUI ${VERSION_TAG_GUI})
-  write_static_version_header(${VERSION_TAG_GUI})
-else()
-  message(STATUS "WARNING: Git was not found!")
-  write_static_version_header("unknown")
-endif ()
 add_custom_target(genversiongui ALL
     DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/version.js")
